src/app.md

```diff
@@ -249,6 +250,62 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-avoid-read-write]: #ex-avoid-read-write
+<a name="ex-avoid-read-write"></a>
+## Avoid writing and reading from a same file
+
+[![same_file-badge]][same_file] [![cat-filesystem-badge]][cat-filesystem]
+
+Use [`same_file::Handle`] to a file that can be tested for equality with
+other handles. In this example, the handles of file to be read from and
+to be written to are tested for equality.
+```bash
+cargo run
+```
+will display the contents of the file if the two files are not same and
+```bash
+cargo run >> ./new.txt
+```
+will display the error (because the two files are same).
+
+```rust,no_run
+# #[macro_use]
+# extern crate error_chain;
+extern crate same_file;
+
+use same_file::Handle;
+use std::path::Path;
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+#
+# error_chain! {
+#     foreign_links {
+#          IOError(::std::io::Error);
+#     }
+# }
+
+fn run() -> Result<()> {
+    let path_to_read = Path::new("new.txt");
+
+    let stdout_handle = Handle::stdout()?;
+    let handle = Handle::from_path(path_to_read)?;
+
+    if stdout_handle == handle {
+        bail!("You are reading and writing to the same file");
+    } else {
+        let file = File::open(&path_to_read)?;
+        let file = BufReader::new(file);
+        for (num, line) in file.lines().enumerate() {
+            println!("{} : {}", num, line?.to_uppercase());
+        }
+    }
+
+    Ok(())
+}
+#
+#quick_main!(run);
```

Comment:
Hi @alisha17 the CI fails due to 
`#quick_main!(run);` instead of `# quick_main!(run);`

Please update the code rebase to the latest master and `git push --force` :smile: